gulpfile.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  'use strict';
+
+  var gulp = require('gulp'),
+      jscs = require('gulp-jscs'),
+      jshint = require('gulp-jshint'),
```

Comment:
I take it `jscs` doesn’t always cover `jshint`-style rules? Only stylistics stuff?